botocore/session.py

```diff
@@ -53,7 +49,9 @@ class Session(object):
     :ivar profile: The current profile.
     """
 
-    AllEvents = {
+    #: A dictionary where each key is an event name and the value
+    #: is the formatting string used to construct a new event.
+    ALL_EVENTS = {
```

Comment:
Yeah, this is interesting because as we started to refactor code to be less dependent on a session, and instead just take event_emitters where necessary, they now no longer have this list of available events accessible.

The original idea behind this was to a) see what events were available and b) avoid typos in `emit()/register()` calls when referencing event names.

What do you think we should do with these now?